sql/hive/src/main/scala/org/apache/spark/sql/hive/MetastoreRelation.scala

```diff
@@ -111,23 +111,8 @@ private[hive] case class MetastoreRelation(
   @transient override lazy val statistics: Statistics = {
     catalogTable.stats.getOrElse(Statistics(
       sizeInBytes = {
-        val totalSize = hiveQlTable.getParameters.get(StatsSetupConst.TOTAL_SIZE)
-        val rawDataSize = hiveQlTable.getParameters.get(StatsSetupConst.RAW_DATA_SIZE)
```

Comment:
In the master branch, we do not use Hive-generated `numRows`... Let me fix it in this PR.